test/app/views/core/SubscribeModal.spec.coffee

```diff
@@ -29,8 +29,6 @@ productList = [
   }
 ]
 
-productListInternational = _.map(productList, (p) -> _.assign({}, p, {name: 'brazil_' + p.name}))
```

Comment:
Let's keep the tests here but just rename from `brazil` to another country.

Reply:
Kept this line and test case where it was used.

Reply:
Let's do `macedonia` instead of `china`, as on the server.